client/pkg/eventlog/log_segment.go

```diff
@@ -203,7 +221,51 @@ func (s *segment) Read(ctx context.Context, from int64, size int16, pollingTimeo
 		}
 		off, ok := v.(int32)
 		if !ok {
-			return events, errors.ErrCorruptedEvent
+			return events, errors.ErrInvalidRequest.WithMessage("corrupted event")
```

Comment:
return `ErrInvalidRequest` is unreasonable

Reply:
error msg can described clearly